### ISSUE TITLE
Markdown for decorative images

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#1327]
meta:290293

We encourage you to provide alt text for images with a little "nag" if you leave it out.  Occasionally, blank alt text is correct per WCAG.  Can we do something to allow authors to say "this image should have no alt text" and have that be accepted?  I don't know if "do something" should be explicit markdown (as suggested on Meta) or something you can click on when you get the nag to say "yes I know and that's correct" and dismiss the nag.
